migrations/Version20240130173300.php

```diff
@@ -24,28 +24,28 @@ protected function getDatabaseType(): ?MSPDatabaseType
     protected function onUp(Schema $schema): void
     {
         $this->addSql(
-            'ALTER TABLE IF EXISTS `geometry` CHANGE geometry_subtractive geometry_subtractive INT(11) NULL'
+            'ALTER TABLE `geometry` CHANGE geometry_subtractive geometry_subtractive INT(11) NULL'
```

Comment:
this is not supported by mariadb 10.4 using by our xampp still

Reply:
ah shit sorry